users/views.py

```diff
@@ -1,3 +1,47 @@
+import json, re, bcrypt
 
+from django.http import JsonResponse
+from django.views import View
+from django.core.exceptions import ValidationError
+
+
+from users.validation import email_check, password_check
+from .models import User
+
+# Create your views here.
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data          = json.loads(request.body)
+            user_name     = data['korean_name']
+            user_email    = data['email']
+            user_password = data['password']
+            user_address  = data['address']
+            user_phone    = data['phone_number']
+
+            email_check(user_email)
+
+            if User.objects.filter(email=user_email).exists(): 
+                return JsonResponse({'MESSAGE' : 'EXISTENT_EMAIL'}, status=400)
```

Comment:
key 부분은 주로 소문자를 사용합니다.
다른 부분들도 찾아서 수정해 주세요.
```suggestion
                return JsonResponse({'message' : 'EXISTENT_EMAIL'}, status=400)
```

Reply:
수정하겠습니다!